packages/core/src/server/environment.ts

```diff
@@ -0,0 +1,76 @@
+import { join } from 'node:path';
+import type { EnvironmentContext, Stats } from '@rsbuild/shared';
+import { run } from './runner';
+
+export type ServerUtils = {
+  readFileSync: (fileName: string) => string;
+  environment: EnvironmentContext;
+};
+
+export const loadBundle = async <T>(
+  stats: Stats,
+  entryName: string,
+  utils: ServerUtils,
+): Promise<T> => {
+  const { chunks, entrypoints, outputPath } = stats.toJson({
+    all: false,
+    chunks: true,
+    entrypoints: true,
+    outputPath: true,
+  });
+
+  if (!entrypoints?.[entryName]) {
+    throw new Error(`can't find ssr entry(${entryName})`);
+  }
+
+  const { chunks: entryChunks } = entrypoints[entryName];
+
+  // find main entryChunk from chunks
+  const files = entryChunks.reduce<string[]>((prev, curr) => {
+    const c = curr
+      ? chunks?.find((c) => c.names.includes(curr) && c.entry)
+      : undefined;
+
+    return c
+      ? prev.concat(c?.files.filter((file) => !file.endsWith('.css')))
+      : prev;
+  }, []);
+
+  if (files.length === 0) {
+    throw new Error(`can't get ssr bundle by entryName(${entryName})`);
+  }
+
+  // An entrypoint should have only one entryChunk, but there may be some boundary cases
+  if (files.length > 1) {
+    throw new Error(
+      `only support load single ssr bundle entry, but got ${files.length}: ${files.join(',')}`,
```

Comment:
What is `bundle entry`

Reply:
Related rsbuild source.entry config, but as far as I know, in the bundle results, only one chunk of every entrypoint has an entry attribute of `true`.